omod/src/main/webapp/manageMappings.jsp

```diff
@@ -27,6 +27,13 @@
 	<form method="POST" enctype="multipart/form-data">
 		<input type="file" name="mapping">
 		<input type="submit" value='<spring:message code="dhisconnector.upload"/>'>
+		<br />
+		<label style="margin-top: 2rem;">
+			<input type="checkbox" value="on" name="shouldReplaceMetadata">
+			<span style="font-size: smaller">
+				Override metadata with matching uuids
```

Comment:
Can we localise this message?

Reply:
I'm sorry, I don't know what it means to localise a message.

Reply:
When we switch the language from English to say French, this text will remain English.

Reply:
@dkayiwa I updated the PR can you please check again and see 🙂